fixtures/context.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+from fixtures.pytest_store import store
+from utils.log import logger
+
+
+@pytest.mark.hookwrapper
+def pytest_pyfunc_call(pyfuncitem):
+    """Inspects and consumes certain exceptions
+
+    The guts of this function are explained above in the module documentation.
+
+    Args:
+        pyfuncitem: A pytest test item.
+    """
+    # do whatever you want before the next hook executes
+    # Login as the "new" user to run the test under
+    logger.info('I AM DOING IT PAPA')
+    if 'context' in pyfuncitem.fixturenames:
+        with store.current_appliance.context.use(pyfuncitem._request.getfuncargvalue('context')):
```

Comment:
sure, there are some places where we need the REAL object, but in this case, that's not it :)